analyzer/src/functTest/java/org/jboss/gm/analyzer/alignment/SimpleProjectFunctionalTest.java

```diff
@@ -211,4 +220,134 @@ public void verifySnapshotHandling() throws IOException, URISyntaxException, Man
             });
         });
     }
+
+    @Test
+    public void verifyAlignmentReportJson() throws IOException, URISyntaxException {
+        final File projectRoot = tempDir.newFolder("simple-project");
+        assertThat(projectRoot).isDirectory();
+        final TestManipulationModel alignmentModel = TestUtils.align(projectRoot, projectRoot.getName());
+        assertThat(alignmentModel).isNotNull();
+        final File jsonFile = new File(projectRoot, Configuration.REPORT_JSON_OUTPUT_FILE);
+        final File textFile = new File(projectRoot, "alignmentReport.txt");
+        assertThat(jsonFile).isFile();
+        assertThat(textFile).doesNotExist();
+        final PME jsonReport = JSONUtils.fileToJSON(jsonFile);
+        final String jsonString = FileUtils.readFileToString(jsonFile, StandardCharsets.UTF_8);
+        assertThat(jsonString).isEqualTo("{\n" +
+                "  \"executionRoot\" : {\n" +
+                "    \"groupId\" : \"org.acme.gradle\",\n" +
+                "    \"artifactId\" : \"root\",\n" +
+                "    \"version\" : \"1.0.1\"\n" +
+                "  },\n" +
```

Comment:
This next gav is appearing twice, since it adds one for the project itself, and then apparently it also appears as a module. Is this correct?

Reply:
As per https://github.com/release-engineering/pom-manipulation-ext/blob/master/common/src/main/java/org/commonjava/maven/ext/common/json/PME.java it was to denote where pme was run from and was used by repour. Its probably with syncing with @thescouser89 to check how repour works with gme in this case.

Reply:
OK, I think the reason it is printed twice here are in the text is that it's treating a module as a dependency?